annotations/component-annotations/src/it/component-feat-275-provide-port/pom.xml

```diff
@@ -9,7 +9,7 @@
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>io.dekorate</groupId>
-	<artifactId>issue-275</artifactId>
+	<artifactId>component-feat-275-provide-port</artifactId>
```

Comment:
Don't have versy strong feelings about it, but I do feel that `component-` is a bit redundant, given that the tests live under `component-annoations`.

Reply:
Yes, good remark. I didn't like it completely but I didn't know why.